src/main/java/org/broadinstitute/hellbender/tools/HaplotypeCallerSpark.java

```diff
@@ -13,6 +13,11 @@
 import org.apache.spark.api.java.JavaSparkContext;
 import org.apache.spark.api.java.function.FlatMapFunction;
 import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.barclay.argparser.*;
```

Comment:
I won't comment on each one, but there are quite a few of these redundant "*" imports the in this PR).